website/src/components/APITable/index.tsx

```diff
@@ -1,27 +1,27 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-
 import React, {
   type ComponentProps,
   type ReactElement,
   type ReactNode,
   isValidElement,
   useRef,
   useEffect,
+  useCallback,
+  memo,
 } from 'react';
 import useBrokenLinks from '@docusaurus/useBrokenLinks';
-import {useHistory} from '@docusaurus/router';
+import { useHistory } from '@docusaurus/router';
```

Comment:
There's no reason to change the formatting here, and to not comply with our standard formatting rules configured in the linter